packages/terra-clinical-modal-manager/src/reducers/modalManager.js

```diff
@@ -0,0 +1,43 @@
+import { open, push, pop, maximize, minimize, defaultState } from './disclosureUtils';
+
+import {
+  OPEN,
+  CLOSE,
+  PUSH,
+  POP,
+  MAXIMIZE,
+  MINIMIZE,
+} from '../actions/modalManager';
+
+const supportedSizes = {
+  tiny: 'tiny',
+  small: 'small',
+  medium: 'medium',
+  large: 'large',
+  huge: 'huge',
+};
+
+const defaultModalState = Object.assign({}, defaultState, {
+  size: supportedSizes.small,
+});
+
+const modalManager = (state = defaultModalState, action) => {
+  switch (action.type) {
+    case OPEN:
+      return Object.assign({}, open(state, action), { size: action.data.size || supportedSizes.small });
```

Comment:
The `size` value is passed through the disclose API into the `open` action creator for the ModalManager.

For the tests, it originates here: https://github.com/cerner/terra-clinical/blob/terra-clinical-modal-manager/packages/terra-clinical-modal-manager/tests/nightwatch/DemoContainer.jsx#L22